### ISSUE TITLE
TypeError in `WelcomePaneBanner.tsx` breaks `@applications/mail` for some reason

### ISSUE THREAD

**Issue by Nekonyx** [ProtonMail/WebClients#255]
If `filteredOptions` is an empty array:
https://github.com/ProtonMail/WebClients/blob/8d7f8a902034405988bd70431c714e9fdbb37a1d/applications/mail/src/app/components/view/WelcomePaneBanner.tsx#L324

Then random element from it throws an exception later:
https://github.com/ProtonMail/WebClients/blob/8d7f8a902034405988bd70431c714e9fdbb37a1d/applications/mail/src/app/components/view/WelcomePaneBanner.tsx#L326
```
TypeError: Cannot read properties of undefined (reading 'id')
    at WelcomePaneBanner.tsx:334
    at WelcomePaneBanner.tsx:339
    at fk (react-dom.production.min.js:262)
    at R.unstable_runWithPriority (scheduler.production.min.js:18)
    at gg (react-dom.production.min.js:122)
    at Oj (react-dom.production.min.js:261)
    at react-dom.production.min.js:261
    at V (scheduler.production.min.js:16)
    at MessagePort._e.port1.onmessage (scheduler.production.min.js:12)
```

![image](https://user-images.githubusercontent.com/16444061/136540813-27d439e1-852a-495c-a538-224be09faa39.png)

**Comment by vladimiry** [comment]
I'd suggest to consider enabling the `compilerOptions.noUncheckedIndexedAccess` flag in the https://github.com/ProtonMail/WebClients/blob/main/tsconfig.base.json file. This would allow squeezing more value out of the TypeScript. So the issue like this would likely be spotted by a compiler as the `randomOption` var will be treated as "possibly undefined".

**Comment by Nekonyx** [comment]
Any updates? I can't check my emails for about a week.
![image](https://user-images.githubusercontent.com/16444061/136926075-156832c6-21d1-41c9-9b07-690675194244.png)

**Comment by mmso** [comment]
Thank you for the report @Nekonyx. We will push something today for this.

_Correction: Probably not today. Tomorrow more likely_

**Comment by Nekonyx** [comment]
Fixed in [9364a28f42605150cffaa936e5abb837cd0cd7b9]
Now it works, thanks 😄
